relay/polkadot/src/lib.rs

```diff
@@ -1548,7 +1548,7 @@ impl pallet_rc_migrator::Config for Runtime {
 	>;
 	type Currency = Balances;
 	type CheckingAccount = xcm_config::CheckAccount;
-	type SendXcm = xcm_config::XcmRouter;
+	type SendXcm = xcm_config::XcmRouterWithoutException;
```

Comment:
Why do we need a special one here on the relay? I thought we want to use the one that disables during migration.